### PR TITLE
Library JavaScript  Millstone One

### PR DESCRIPTION
**Pull request by Berabjesus**
### Tools and Techs used
- JavaScript
- HTML and CSS
- Bootstrap
- Eslint

### CODE REVIEW

File: js/dom.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
+/* eslint-disable func-names */
+/* eslint-disable no-undef */
+/* eslint-disable no-loop-func */
```
Comment (lucasmazo23) [inline js/dom.js]: - [x] [OPTIONAL] Right now there's a lot of `disable linter` lines, remember that Eslint rules are meant to catch bad code and enforce best practices. 
image

**Review by lucasmazo23 — CHANGES_REQUESTED** [review]
## Changes required 🔧
### You are doing a great job so far, and the project is looking good.
 
### For the project to get approved kindly implement these changes:
 
### Required changes:
- [x] [REQUIRED] The README file need some love, right now it's empty D:
image
[Source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
- [x] [REQUIRED] When I click on add a new book, with the checkbox unchecked it gives me the incorrect result
checkbox
- [x] [REQUIRED] your .lock files should be in the repository, right now there's none
image
image
### Optional changes:
- [x] [OPTIONAL] Right now there's a lot of `disable linter` lines, remember that Eslint rules are meant to catch bad code and enforce best practices. 
image

### Happy coding! 💻
## Lucas Mazo

File: js/dom.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
```
Comment (kubilaycaglayan) [inline js/dom.js]: Try to avoid disabling the eslint checks for the whole file.

**Review by kubilaycaglayan — CHANGES_REQUESTED** [review]
#### Hello Team 👋
#### Status: Changes Required ♻️

Great job so far! 👏👋
Thanks for making the changes required by the previous TSE.
Let's cover a few details together to make your project better 😊⏬

### ESLint

- [ ] We should not overuse ESLint disable comments

💡 Currently in this project we have three different disables for the whole files. You can go over these documentation to see the good and bad code examples. I want you to give a try to fix these without disabling the linters 🙏😊

https://eslint.org/docs/rules/no-use-before-define
https://eslint.org/docs/rules/no-unused-vars
https://eslint.org/docs/rules/no-undef

## That's it, you can submit another code review request after making these changes ♻️😇

With all my respect 🙏
> Kubilay Caglayan | Technical Support Engineer

File: js/dom.js (hunk 2 above)
Comment (Berabjesus) [reply js/dom.js]: With out the '/* eslint-disable no-unused-vars */' the renderBook() function will give an error since its only called from the index.js

Comment (kubilaycaglayan) [reply js/dom.js]: I see, when you have a situation that there's no other choice than disabling an eslint rule please do it for only a specific line. Can you try this?

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @Emmanuelaaron and @Berabjesus :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**